src/utility/copy.cpp

```diff
@@ -108,7 +131,12 @@ DuckdbCopy(PlannedStmt *pstmt, const char *query_string, struct QueryEnvironment
 		raw_stmt->stmt_location = pstmt->stmt_location;
 		raw_stmt->stmt_len = pstmt->stmt_len;
 
-		rewritten = pg_analyze_and_rewrite_fixedparams(raw_stmt, query_string, NULL, 0, NULL);
+		#if PG_VERSION_NUM >= 150000
+			rewritten = pg_analyze_and_rewrite_fixedparams(raw_stmt, query_string, NULL, 0, NULL);
+		#else
+			rewritten = pg_analyze_and_rewrite(raw_stmt, query_string, NULL, 0, NULL);
+		#endif
```

Comment:
I guess this change is only needed for PG14 support. Let's remove it from this PR for now.

Reply:
Ok, I will submit this paragraph again at PG14